Cannot configure Registry with UAA

##### **Overview of the issue**

Again I cannot run Registry with UAA without errors. Registry is just cloned from GitHub, UAA is a new generated application (I only changed the password to the database).  
Logs of Registry start:

```
2017-02-17 16:39:38.868  INFO 3360 --- [  restartedMain] i.g.jhipster.registry.JHipsterRegistry   : The following profiles are active: native,dev
2017-02-17 16:39:46.781  WARN 3360 --- [  restartedMain] o.s.c.a.ConfigurationClassPostProcessor  : Cannot enhance @Configuration bean definition 'refreshScope' since its singleton instance has been created too early. The typical cause is a non-static @Bean method with a BeanDefinitionRegistryPostProcessor return type: Consider declaring such methods as 'static'.
2017-02-17 16:39:48.943 DEBUG 3360 --- [  restartedMain] i.g.j.r.config.MetricsConfiguration      : Registering JVM gauges
2017-02-17 16:39:49.501 DEBUG 3360 --- [  restartedMain] i.g.j.r.config.MetricsConfiguration      : Initializing Metrics JMX reporting
2017-02-17 16:39:52.905  INFO 3360 --- [  restartedMain] org.xnio                                 : XNIO version 3.3.6.Final
2017-02-17 16:39:53.013  INFO 3360 --- [  restartedMain] org.xnio.nio                             : XNIO NIO Implementation Version 3.3.6.Final
2017-02-17 16:39:57.144  INFO 3360 --- [  restartedMain] i.g.j.registry.config.WebConfigurer      : Web application configuration, using profiles: [native, dev]
2017-02-17 16:39:57.232 DEBUG 3360 --- [  restartedMain] i.g.j.registry.config.WebConfigurer      : Initializing Metrics registries
2017-02-17 16:39:57.316 DEBUG 3360 --- [  restartedMain] i.g.j.registry.config.WebConfigurer      : Registering Metrics Filter
2017-02-17 16:39:57.317 DEBUG 3360 --- [  restartedMain] i.g.j.registry.config.WebConfigurer      : Registering Metrics Servlet
2017-02-17 16:39:57.500  INFO 3360 --- [  restartedMain] i.g.j.registry.config.WebConfigurer      : Web application fully configured
2017-02-17 16:39:57.725  INFO 3360 --- [  restartedMain] i.g.jhipster.registry.JHipsterRegistry   : Running with Spring profile(s) : [native, dev]
2017-02-17 16:40:05.055  INFO 3360 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : Client configured to neither register nor query for data.
2017-02-17 16:40:05.140  INFO 3360 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : Discovery Client initialized at timestamp 1487342405139 with initial instances count: 0
2017-02-17 16:40:05.945  INFO 3360 --- [  restartedMain] c.n.eureka.DefaultEurekaServerContext    : Initializing ...
2017-02-17 16:40:06.165  INFO 3360 --- [  restartedMain] c.n.eureka.cluster.PeerEurekaNodes       : Adding new peer nodes [http://localhost:8761/eureka/]
2017-02-17 16:40:07.777  INFO 3360 --- [  restartedMain] c.n.d.provider.DiscoveryJerseyProvider   : Using JSON encoding codec LegacyJacksonJson
2017-02-17 16:40:07.925  INFO 3360 --- [  restartedMain] c.n.d.provider.DiscoveryJerseyProvider   : Using JSON decoding codec LegacyJacksonJson
2017-02-17 16:40:08.011  INFO 3360 --- [  restartedMain] c.n.d.provider.DiscoveryJerseyProvider   : Using XML encoding codec XStreamXml
2017-02-17 16:40:08.059  INFO 3360 --- [  restartedMain] c.n.d.provider.DiscoveryJerseyProvider   : Using XML decoding codec XStreamXml
2017-02-17 16:40:09.249  INFO 3360 --- [  restartedMain] c.n.eureka.cluster.PeerEurekaNodes       : Replica node URL:  http://localhost:8761/eureka/
2017-02-17 16:40:09.401  INFO 3360 --- [  restartedMain] c.n.e.registry.AbstractInstanceRegistry  : Finished initializing remote region registries. All known remote regions: []
2017-02-17 16:40:09.451  INFO 3360 --- [  restartedMain] c.n.eureka.DefaultEurekaServerContext    : Initialized
2017-02-17 16:40:16.182  WARN 3360 --- [  restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2017-02-17 16:40:16.362  INFO 3360 --- [  restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2017-02-17 16:40:16.396  WARN 3360 --- [  restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2017-02-17 16:40:16.409  INFO 3360 --- [  restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2017-02-17 16:40:18.002  INFO 3360 --- [      Thread-14] c.n.e.r.PeerAwareInstanceRegistryImpl    : Got 1 instances from neighboring DS node
2017-02-17 16:40:18.065  INFO 3360 --- [  restartedMain] i.g.jhipster.registry.JHipsterRegistry   : Started JHipsterRegistry in 49.049 seconds (JVM running for 54.247)
2017-02-17 16:40:18.082  INFO 3360 --- [  restartedMain] i.g.jhipster.registry.JHipsterRegistry   : 
----------------------------------------------------------
	Application 'jhipster-registry' is running! Access URLs:
	Local: 		http://127.0.0.1:8761
	External: 	http://192.168.1.31:8761
	Profile(s): 	[native, dev]
----------------------------------------------------------
2017-02-17 16:40:18.085  INFO 3360 --- [      Thread-14] c.n.e.r.PeerAwareInstanceRegistryImpl    : Renew threshold is: 1
2017-02-17 16:40:18.098  INFO 3360 --- [      Thread-14] c.n.e.r.PeerAwareInstanceRegistryImpl    : Changing status to UP
```
Logs of UAA start:

```
2017-02-17 16:40:47.328 DEBUG 1272 --- [  restartedMain] s.n.www.protocol.http.HttpURLConnection  : sun.net.www.MessageHeader@ad296f66 pairs: {GET /config/sample/dev/master HTTP/1.1: null}{Accept: application/json, application/*+json}{Authorization: Basic YWRtaW46YWRtaW4=}{User-Agent: Java/1.8.0_121}{Host: localhost:8761}{Connection: keep-alive}
2017-02-17 16:40:49.162 DEBUG 1272 --- [  restartedMain] s.n.www.protocol.http.HttpURLConnection  : sun.net.www.MessageHeader@2193a9b511 pairs: {null: HTTP/1.1 200 OK}{Expires: 0}{Cache-Control: no-cache, no-store, max-age=0, must-revalidate}{X-XSS-Protection: 1; mode=block}{Pragma: no-cache}{Date: Fri, 17 Feb 2017 14:40:48 GMT}{Connection: keep-alive}{X-Content-Type-Options: nosniff}{Transfer-Encoding: chunked}{Content-Type: application/json;charset=UTF-8}{X-Application-Context: jhipster-registry:native,dev:8761}
2017-02-17 16:40:49.458  INFO 1272 --- [  restartedMain] io.delverdev.microservices.SampleApp     : The following profiles are active: swagger,dev
2017-02-17 16:41:03.791  WARN 1272 --- [  restartedMain] o.s.c.a.ConfigurationClassPostProcessor  : Cannot enhance @Configuration bean definition 'refreshScope' since its singleton instance has been created too early. The typical cause is a non-static @Bean method with a BeanDefinitionRegistryPostProcessor return type: Consider declaring such methods as 'static'.
2017-02-17 16:41:06.527 DEBUG 1272 --- [  restartedMain] i.d.m.config.AsyncConfiguration          : Creating Async Task Executor
2017-02-17 16:41:11.393 DEBUG 1272 --- [  restartedMain] i.d.m.config.MetricsConfiguration        : Registering JVM gauges
2017-02-17 16:41:11.951 DEBUG 1272 --- [  restartedMain] i.d.m.config.MetricsConfiguration        : Monitoring the datasource
2017-02-17 16:41:12.063 DEBUG 1272 --- [  restartedMain] i.d.m.config.MetricsConfiguration        : Initializing Metrics JMX reporting
2017-02-17 16:41:15.457 DEBUG 1272 --- [  restartedMain] i.d.m.config.CacheConfiguration          : Configuring Hazelcast
2017-02-17 16:41:15.694 DEBUG 1272 --- [  restartedMain] i.d.m.config.CacheConfiguration          : Configuring Hazelcast clustering for instanceId: sample
2017-02-17 16:41:15.799 DEBUG 1272 --- [  restartedMain] i.d.m.config.CacheConfiguration          : Application is running with the "dev" profile, Hazelcast cluster will only work with localhost instances
2017-02-17 16:41:18.399  INFO 1272 --- [  restartedMain] c.n.d.provider.DiscoveryJerseyProvider   : Using JSON encoding codec LegacyJacksonJson
2017-02-17 16:41:18.435  INFO 1272 --- [  restartedMain] c.n.d.provider.DiscoveryJerseyProvider   : Using JSON decoding codec LegacyJacksonJson
2017-02-17 16:41:19.201  INFO 1272 --- [  restartedMain] c.n.d.provider.DiscoveryJerseyProvider   : Using XML encoding codec XStreamXml
2017-02-17 16:41:19.258  INFO 1272 --- [  restartedMain] c.n.d.provider.DiscoveryJerseyProvider   : Using XML decoding codec XStreamXml
2017-02-17 16:41:20.212  INFO 1272 --- [  restartedMain] c.n.d.s.r.aws.ConfigClusterResolver      : Resolving eureka endpoints via configuration
2017-02-17 16:41:20.241 DEBUG 1272 --- [  restartedMain] c.n.discovery.endpoint.EndpointUtils     : The availability zone for the given region us-east-1 are [defaultZone]
2017-02-17 16:41:20.245 DEBUG 1272 --- [  restartedMain] c.n.d.s.r.aws.ConfigClusterResolver      : Config resolved to [AwsEndpoint{ serviceUrl='http://admin:admin@localhost:8761/eureka/', region='us-east-1', zone='defaultZone'}]
2017-02-17 16:41:20.290 DEBUG 1272 --- [  restartedMain] c.n.d.s.r.a.ZoneAffinityClusterResolver  : Local zone=defaultZone; resolved to: [AwsEndpoint{ serviceUrl='http://admin:admin@localhost:8761/eureka/', region='us-east-1', zone='defaultZone'}]
2017-02-17 16:41:20.354  INFO 1272 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : Disable delta property : false
2017-02-17 16:41:20.361  INFO 1272 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : Single vip registry refresh property : null
2017-02-17 16:41:20.361  INFO 1272 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : Force full registry fetch : false
2017-02-17 16:41:20.361  INFO 1272 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : Application is null : false
2017-02-17 16:41:20.362  INFO 1272 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : Registered Applications size is zero : true
2017-02-17 16:41:20.368  INFO 1272 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : Application version is -1: true
2017-02-17 16:41:20.368  INFO 1272 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : Getting all instance registry info from the eureka server
2017-02-17 16:41:20.373 DEBUG 1272 --- [  restartedMain] c.n.d.s.t.d.SessionedEurekaHttpClient    : Ending a session and starting anew
2017-02-17 16:41:20.394 DEBUG 1272 --- [  restartedMain] n.d.s.t.j.AbstractJerseyEurekaHttpClient : Created client for url: http://admin:admin@localhost:8761/eureka/
2017-02-17 16:41:20.716 DEBUG 1272 --- [  restartedMain] c.n.d.shared.MonitoredConnectionManager  : Get connection: {}->http://localhost:8761, timeout = 5000
2017-02-17 16:41:20.718 DEBUG 1272 --- [  restartedMain] c.n.d.shared.NamedConnectionPool         : [{}->http://localhost:8761] total kept alive: 0, total issued: 0, total allocated: 0 out of 200
2017-02-17 16:41:20.726 DEBUG 1272 --- [  restartedMain] c.n.d.shared.NamedConnectionPool         : No free connections [{}->http://localhost:8761][null]
2017-02-17 16:41:20.729 DEBUG 1272 --- [  restartedMain] c.n.d.shared.NamedConnectionPool         : Available capacity: 50 out of 50 [{}->http://localhost:8761][null]
2017-02-17 16:41:20.743 DEBUG 1272 --- [  restartedMain] c.n.d.shared.NamedConnectionPool         : Creating new connection [{}->http://localhost:8761]
2017-02-17 16:41:23.887 DEBUG 1272 --- [  restartedMain] c.n.d.shared.MonitoredConnectionManager  : Released connection is reusable.
2017-02-17 16:41:23.898 DEBUG 1272 --- [  restartedMain] c.n.d.shared.NamedConnectionPool         : Releasing connection [{}->http://localhost:8761][null]
2017-02-17 16:41:23.898 DEBUG 1272 --- [  restartedMain] c.n.d.shared.NamedConnectionPool         : Pooling connection [{}->http://localhost:8761][null]; keep alive indefinitely
2017-02-17 16:41:23.909 DEBUG 1272 --- [  restartedMain] c.n.d.shared.NamedConnectionPool         : Notifying no-one, there are no waiting threads
2017-02-17 16:41:23.919 DEBUG 1272 --- [  restartedMain] n.d.s.t.j.AbstractJerseyEurekaHttpClient : Jersey HTTP GET http://admin:admin@localhost:8761/eureka//apps/?; statusCode=200
2017-02-17 16:41:23.920 DEBUG 1272 --- [  restartedMain] c.n.d.s.t.d.RedirectingEurekaHttpClient  : Pinning to endpoint null
2017-02-17 16:41:23.920  INFO 1272 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : The response status is 200
2017-02-17 16:41:23.930 DEBUG 1272 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : Got full registry with apps hashcode 
2017-02-17 16:41:23.931 DEBUG 1272 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : The total number of all instances in the client now is 0
2017-02-17 16:41:23.936  INFO 1272 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : Starting heartbeat executor: renew interval is: 30
2017-02-17 16:41:23.943  INFO 1272 --- [  restartedMain] c.n.discovery.InstanceInfoReplicator     : InstanceInfoReplicator onDemand update allowed rate per min is 4
2017-02-17 16:41:23.956  INFO 1272 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : Discovery Client initialized at timestamp 1487342483956 with initial instances count: 0
2017-02-17 16:41:24.667  INFO 1272 --- [  restartedMain] c.h.instance.DefaultAddressPicker        : [LOCAL] [dev] [3.7] Picked [127.0.0.1]:15700, using socket ServerSocket[addr=/0:0:0:0:0:0:0:0,localport=15700], bind any local is true
2017-02-17 16:41:24.762  INFO 1272 --- [  restartedMain] com.hazelcast.system                     : [127.0.0.1]:15700 [dev] [3.7] Hazelcast 3.7 (20160817 - 1302600) starting at [127.0.0.1]:15700
2017-02-17 16:41:24.769  INFO 1272 --- [  restartedMain] com.hazelcast.system                     : [127.0.0.1]:15700 [dev] [3.7] Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
2017-02-17 16:41:24.778  INFO 1272 --- [  restartedMain] com.hazelcast.system                     : [127.0.0.1]:15700 [dev] [3.7] Configured Hazelcast Serialization version : 1
2017-02-17 16:41:25.618  INFO 1272 --- [  restartedMain] c.h.s.i.o.impl.BackpressureRegulator     : [127.0.0.1]:15700 [dev] [3.7] Backpressure is disabled
2017-02-17 16:41:29.398 DEBUG 1272 --- [  restartedMain] c.h.internal.cluster.ClusterService      : [127.0.0.1]:15700 [dev] [3.7] Updating members [Member [127.0.0.1]:15700 - bbc34649-e057-499f-8c46-7948e507ecd7 this]
2017-02-17 16:41:29.404 DEBUG 1272 --- [  restartedMain] c.h.i.p.InternalPartitionService         : [127.0.0.1]:15700 [dev] [3.7] Adding Member [127.0.0.1]:15700 - bbc34649-e057-499f-8c46-7948e507ecd7 this
2017-02-17 16:41:29.483  INFO 1272 --- [  restartedMain] com.hazelcast.instance.Node              : [127.0.0.1]:15700 [dev] [3.7] Creating TcpIpJoiner
2017-02-17 16:41:29.490  INFO 1272 --- [  restartedMain] com.hazelcast.core.LifecycleService      : [127.0.0.1]:15700 [dev] [3.7] [127.0.0.1]:15700 is STARTING
2017-02-17 16:41:29.852  INFO 1272 --- [  restartedMain] c.h.s.i.o.impl.OperationExecutorImpl     : [127.0.0.1]:15700 [dev] [3.7] Starting 2 partition threads
2017-02-17 16:41:29.852  INFO 1272 --- [  restartedMain] c.h.s.i.o.impl.OperationExecutorImpl     : [127.0.0.1]:15700 [dev] [3.7] Starting 3 generic threads (1 dedicated for priority tasks)
2017-02-17 16:41:29.871  INFO 1272 --- [  restartedMain] c.h.n.t.n.NonBlockingIOThreadingModel    : [127.0.0.1]:15700 [dev] [3.7] TcpIpConnectionManager configured with Non Blocking IO-threading model: 3 input threads and 3 output threads
2017-02-17 16:41:29.872 DEBUG 1272 --- [  restartedMain] c.h.n.t.n.NonBlockingIOThreadingModel    : [127.0.0.1]:15700 [dev] [3.7] IO threads selector mode is SELECT
2017-02-17 16:41:29.936 DEBUG 1272 --- [  restartedMain] com.hazelcast.cluster.impl.TcpIpJoiner   : [127.0.0.1]:15700 [dev] [3.7] This node will assume master role since no possible member where connected to.
2017-02-17 16:41:29.937 DEBUG 1272 --- [  restartedMain] com.hazelcast.cluster.impl.TcpIpJoiner   : [127.0.0.1]:15700 [dev] [3.7] PostJoin master: [127.0.0.1]:15700, isMaster: true
2017-02-17 16:41:29.943  INFO 1272 --- [  restartedMain] com.hazelcast.cluster.impl.TcpIpJoiner   : [127.0.0.1]:15700 [dev] [3.7] 


Members [1] {
	Member [127.0.0.1]:15700 - bbc34649-e057-499f-8c46-7948e507ecd7 this
}

2017-02-17 16:41:30.003  INFO 1272 --- [  restartedMain] com.hazelcast.core.LifecycleService      : [127.0.0.1]:15700 [dev] [3.7] [127.0.0.1]:15700 is STARTED
2017-02-17 16:41:30.396 DEBUG 1272 --- [cached.thread-2] s.n.www.protocol.http.HttpURLConnection  : sun.net.www.MessageHeader@520788285 pairs: {GET /ping?version=3.7&m=bbc34649-e057-499f-8c46-7948e507ecd7&e=false&l=NULL&p=maven&c=1b2fc32a-2cbc-4045-b46a-a4953926c8fb&crsz=A&cssz=A&hdgb=0&ccpp=0&cdn=0&cjv=0&cuptm=66&nuptm=62250&jvmn=Java+HotSpot%28TM%29+64-Bit+Server+VM&jvmv=1.8.0_121&osn=Windows+10&osa=amd64&osv=10.0 HTTP/1.1: null}{User-Agent: Mozilla/5.0}{Host: phonehome.hazelcast.com}{Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2}{Connection: keep-alive}
2017-02-17 16:41:30.541 DEBUG 1272 --- [cached.thread-2] s.n.www.protocol.http.HttpURLConnection  : sun.net.www.MessageHeader@75ec55cf5 pairs: {null: HTTP/1.1 200 OK}{Server: Apache-Coyote/1.1}{Content-Type: text/plain;charset=UTF-8}{Transfer-Encoding: chunked}{Date: Fri, 17 Feb 2017 14:41:14 GMT}
2017-02-17 16:41:33.180  INFO 1272 --- [  restartedMain] i.d.microservices.config.WebConfigurer   : Web application configuration, using profiles: swagger
2017-02-17 16:41:33.190 DEBUG 1272 --- [  restartedMain] i.d.microservices.config.WebConfigurer   : Initializing Metrics registries
2017-02-17 16:41:33.210 DEBUG 1272 --- [  restartedMain] i.d.microservices.config.WebConfigurer   : Registering Metrics Filter
2017-02-17 16:41:33.212 DEBUG 1272 --- [  restartedMain] i.d.microservices.config.WebConfigurer   : Registering Metrics Servlet
2017-02-17 16:41:33.232  INFO 1272 --- [  restartedMain] i.d.microservices.config.WebConfigurer   : Web application fully configured
2017-02-17 16:41:34.036 DEBUG 1272 --- [  restartedMain] i.d.m.config.DatabaseConfiguration       : Configuring Liquibase
2017-02-17 16:41:34.405  WARN 1272 --- [mple-Executor-1] i.g.j.c.liquibase.AsyncSpringLiquibase   : Starting Liquibase asynchronously, your database might not be ready at startup!
2017-02-17 16:41:45.149  INFO 1272 --- [  restartedMain] c.h.h.HazelcastCacheRegionFactory        : Starting up HazelcastCacheRegionFactory
2017-02-17 16:41:45.301  INFO 1272 --- [  restartedMain] com.hazelcast.config.XmlConfigLocator    : Loading 'hazelcast-default.xml' from classpath.
2017-02-17 16:41:46.433 DEBUG 1272 --- [mple-Executor-1] i.g.j.c.liquibase.AsyncSpringLiquibase   : Started Liquibase in 12028 ms
2017-02-17 16:41:50.169 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.MonitoredConnectionManager  : Closing connections idle longer than 30000 SECONDS
2017-02-17 16:41:50.203 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.MonitoredConnectionManager  : Closing connections idle longer than -1 SECONDS
2017-02-17 16:41:50.289 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.NamedConnectionPool         : Closing connections idle longer than 30000 SECONDS
2017-02-17 16:41:50.350 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.NamedConnectionPool         : Closing connections idle longer than 0 SECONDS
2017-02-17 16:41:50.552 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.NamedConnectionPool         : Closing connection last used @ Fri Feb 17 16:41:23 EET 2017
2017-02-17 16:41:50.556 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.NamedConnectionPool         : Deleting connection [{}->http://localhost:8761][null]
2017-02-17 16:41:54.089 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.s.t.d.SessionedEurekaHttpClient    : Ending a session and starting anew
2017-02-17 16:41:54.116  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Disable delta property : false
2017-02-17 16:41:54.119  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Single vip registry refresh property : null
2017-02-17 16:41:54.119  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Force full registry fetch : false
2017-02-17 16:41:54.159 DEBUG 1272 --- [tbeatExecutor-0] n.d.s.t.j.AbstractJerseyEurekaHttpClient : Created client for url: http://admin:admin@localhost:8761/eureka/
2017-02-17 16:41:54.174  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Application is null : false
2017-02-17 16:41:54.174  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Registered Applications size is zero : true
2017-02-17 16:41:54.174  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Application version is -1: false
2017-02-17 16:41:54.190 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.MonitoredConnectionManager  : Get connection: {}->http://localhost:8761, timeout = 5000
2017-02-17 16:41:54.195 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : [{}->http://localhost:8761] total kept alive: 0, total issued: 0, total allocated: 0 out of 200
2017-02-17 16:41:54.195 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : No free connections [{}->http://localhost:8761][null]
2017-02-17 16:41:54.196 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Available capacity: 50 out of 50 [{}->http://localhost:8761][null]
2017-02-17 16:41:54.196 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Creating new connection [{}->http://localhost:8761]
2017-02-17 16:41:54.214  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Getting all instance registry info from the eureka server
2017-02-17 16:41:54.215 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.MonitoredConnectionManager  : Get connection: {}->http://localhost:8761, timeout = 5000
2017-02-17 16:41:54.215 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.NamedConnectionPool         : [{}->http://localhost:8761] total kept alive: 0, total issued: 1, total allocated: 1 out of 200
2017-02-17 16:41:54.224 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.NamedConnectionPool         : No free connections [{}->http://localhost:8761][null]
2017-02-17 16:41:54.224 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.NamedConnectionPool         : Available capacity: 49 out of 50 [{}->http://localhost:8761][null]
2017-02-17 16:41:54.224 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.NamedConnectionPool         : Creating new connection [{}->http://localhost:8761]
2017-02-17 16:41:54.432 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.MonitoredConnectionManager  : Released connection is reusable.
2017-02-17 16:41:54.432 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.NamedConnectionPool         : Releasing connection [{}->http://localhost:8761][null]
2017-02-17 16:41:54.432 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.NamedConnectionPool         : Pooling connection [{}->http://localhost:8761][null]; keep alive indefinitely
2017-02-17 16:41:54.433 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.NamedConnectionPool         : Notifying no-one, there are no waiting threads
2017-02-17 16:41:54.433 DEBUG 1272 --- [freshExecutor-0] n.d.s.t.j.AbstractJerseyEurekaHttpClient : Jersey HTTP GET http://admin:admin@localhost:8761/eureka//apps/?; statusCode=200
2017-02-17 16:41:54.433  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : The response status is 200
2017-02-17 16:41:54.433 DEBUG 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Got full registry with apps hashcode 
2017-02-17 16:41:54.433 DEBUG 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : The total number of all instances in the client now is 0
2017-02-17 16:41:54.636 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.MonitoredConnectionManager  : Released connection is reusable.
2017-02-17 16:41:54.637 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Releasing connection [{}->http://localhost:8761][null]
2017-02-17 16:41:54.637 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Pooling connection [{}->http://localhost:8761][null]; keep alive indefinitely
2017-02-17 16:41:54.637 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Notifying no-one, there are no waiting threads
2017-02-17 16:41:54.638 DEBUG 1272 --- [tbeatExecutor-0] n.d.s.t.j.AbstractJerseyEurekaHttpClient : Jersey HTTP PUT http://admin:admin@localhost:8761/eureka//apps/SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5; statusCode=404
2017-02-17 16:41:54.772 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.s.t.d.RedirectingEurekaHttpClient  : Pinning to endpoint null
2017-02-17 16:41:54.773 DEBUG 1272 --- [tbeatExecutor-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5 - Heartbeat status: 404
2017-02-17 16:41:54.773  INFO 1272 --- [tbeatExecutor-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5 - Re-registering apps/SAMPLE
2017-02-17 16:41:54.773  INFO 1272 --- [tbeatExecutor-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5: registering service...
2017-02-17 16:41:54.781 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.MonitoredConnectionManager  : Get connection: {}->http://localhost:8761, timeout = 5000
2017-02-17 16:41:54.781 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : [{}->http://localhost:8761] total kept alive: 2, total issued: 0, total allocated: 2 out of 200
2017-02-17 16:41:54.781 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Getting free connection [{}->http://localhost:8761][null]
2017-02-17 16:41:55.670 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.MonitoredConnectionManager  : Released connection is reusable.
2017-02-17 16:41:55.748 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Releasing connection [{}->http://localhost:8761][null]
2017-02-17 16:41:55.748 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Pooling connection [{}->http://localhost:8761][null]; keep alive indefinitely
2017-02-17 16:41:55.749 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Notifying no-one, there are no waiting threads
2017-02-17 16:41:55.750 DEBUG 1272 --- [tbeatExecutor-0] n.d.s.t.j.AbstractJerseyEurekaHttpClient : Jersey HTTP POST http://admin:admin@localhost:8761/eureka//apps/SAMPLE with instance sample:01d89341053c58b21bce7ef55cdaa8e5; statusCode=204
2017-02-17 16:41:55.994  INFO 1272 --- [tbeatExecutor-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5 - registration status: 204
2017-02-17 16:41:56.946 DEBUG 1272 --- [  restartedMain] i.d.m.config.CacheConfiguration          : Starting HazelcastCacheManager
2017-02-17 16:41:59.578 DEBUG 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Completed cache refresh task for discovery. All Apps hash code is Local region apps hashcode: , is fetching remote regions? false 
2017-02-17 16:42:01.686 DEBUG 1272 --- [  restartedMain] i.g.j.c.apidoc.SwaggerConfiguration      : Starting Swagger
2017-02-17 16:42:01.766 DEBUG 1272 --- [  restartedMain] i.g.j.c.apidoc.SwaggerConfiguration      : Started Swagger in 29 ms
2017-02-17 16:42:03.955  INFO 1272 --- [nfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5: registering service...
2017-02-17 16:42:04.054 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.MonitoredConnectionManager  : Get connection: {}->http://localhost:8761, timeout = 5000
2017-02-17 16:42:04.054 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.NamedConnectionPool         : [{}->http://localhost:8761] total kept alive: 2, total issued: 0, total allocated: 2 out of 200
2017-02-17 16:42:04.112 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.NamedConnectionPool         : Getting free connection [{}->http://localhost:8761][null]
2017-02-17 16:42:05.289 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.MonitoredConnectionManager  : Released connection is reusable.
2017-02-17 16:42:05.391  WARN 1272 --- [  restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2017-02-17 16:42:05.393  INFO 1272 --- [  restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2017-02-17 16:42:05.394 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.NamedConnectionPool         : Releasing connection [{}->http://localhost:8761][null]
2017-02-17 16:42:05.421 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.NamedConnectionPool         : Pooling connection [{}->http://localhost:8761][null]; keep alive indefinitely
2017-02-17 16:42:05.454  WARN 1272 --- [  restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2017-02-17 16:42:05.455  INFO 1272 --- [  restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2017-02-17 16:42:05.455 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.NamedConnectionPool         : Notifying no-one, there are no waiting threads
2017-02-17 16:42:05.490 DEBUG 1272 --- [nfoReplicator-0] n.d.s.t.j.AbstractJerseyEurekaHttpClient : Jersey HTTP POST http://admin:admin@localhost:8761/eureka//apps/SAMPLE with instance sample:01d89341053c58b21bce7ef55cdaa8e5; statusCode=204
2017-02-17 16:42:05.491  INFO 1272 --- [nfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5 - registration status: 204
2017-02-17 16:42:07.085  INFO 1272 --- [  restartedMain] com.netflix.discovery.DiscoveryClient    : Saw local status change event StatusChangeEvent [timestamp=1487342527085, current=UP, previous=STARTING]
2017-02-17 16:42:07.095 DEBUG 1272 --- [nfoReplicator-0] c.n.discovery.InstanceInfoReplicator     : Executing on-demand update of local InstanceInfo
2017-02-17 16:42:07.096 DEBUG 1272 --- [nfoReplicator-0] c.n.discovery.InstanceInfoReplicator     : Canceling the latest scheduled update, it will be rescheduled at the end of on demand update
2017-02-17 16:42:07.206 DEBUG 1272 --- [nfoReplicator-0] s.n.www.protocol.http.HttpURLConnection  : sun.net.www.MessageHeader@55f95b1c6 pairs: {GET /config/application/dev/master HTTP/1.1: null}{Accept: application/json, application/*+json}{Authorization: Basic YWRtaW46YWRtaW4=}{User-Agent: Java/1.8.0_121}{Host: localhost:8761}{Connection: keep-alive}
2017-02-17 16:42:10.356 DEBUG 1272 --- [nfoReplicator-0] s.n.www.protocol.http.HttpURLConnection  : sun.net.www.MessageHeader@5bff6e2211 pairs: {null: HTTP/1.1 200 OK}{Expires: 0}{Cache-Control: no-cache, no-store, max-age=0, must-revalidate}{X-XSS-Protection: 1; mode=block}{Pragma: no-cache}{Date: Fri, 17 Feb 2017 14:42:10 GMT}{Connection: keep-alive}{X-Content-Type-Options: nosniff}{Transfer-Encoding: chunked}{Content-Type: application/json;charset=UTF-8}{X-Application-Context: jhipster-registry:native,dev:8761}
2017-02-17 16:42:11.460  INFO 1272 --- [nfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5: registering service...
2017-02-17 16:42:11.889 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.MonitoredConnectionManager  : Get connection: {}->http://localhost:8761, timeout = 5000
2017-02-17 16:42:11.942 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.NamedConnectionPool         : [{}->http://localhost:8761] total kept alive: 2, total issued: 0, total allocated: 2 out of 200
2017-02-17 16:42:11.965 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.NamedConnectionPool         : Getting free connection [{}->http://localhost:8761][null]
2017-02-17 16:42:12.281 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.MonitoredConnectionManager  : Released connection is reusable.
2017-02-17 16:42:12.281 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.NamedConnectionPool         : Releasing connection [{}->http://localhost:8761][null]
2017-02-17 16:42:12.281 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.NamedConnectionPool         : Pooling connection [{}->http://localhost:8761][null]; keep alive indefinitely
2017-02-17 16:42:12.281 DEBUG 1272 --- [nfoReplicator-0] c.n.d.shared.NamedConnectionPool         : Notifying no-one, there are no waiting threads
2017-02-17 16:42:12.375 DEBUG 1272 --- [nfoReplicator-0] n.d.s.t.j.AbstractJerseyEurekaHttpClient : Jersey HTTP POST http://admin:admin@localhost:8761/eureka//apps/SAMPLE with instance sample:01d89341053c58b21bce7ef55cdaa8e5; statusCode=204
2017-02-17 16:42:12.375  INFO 1272 --- [nfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5 - registration status: 204
2017-02-17 16:42:12.375 DEBUG 1272 --- [nfoReplicator-0] c.n.discovery.InstanceInfoReplicator     : Executing on-demand update of local InstanceInfo
2017-02-17 16:42:12.375 DEBUG 1272 --- [nfoReplicator-0] c.n.discovery.InstanceInfoReplicator     : Canceling the latest scheduled update, it will be rescheduled at the end of on demand update
2017-02-17 16:42:12.663  INFO 1272 --- [  restartedMain] io.delverdev.microservices.SampleApp     : Started SampleApp in 95.545 seconds (JVM running for 104.907)
2017-02-17 16:42:12.696  INFO 1272 --- [  restartedMain] io.delverdev.microservices.SampleApp     : 
----------------------------------------------------------
	Application 'sample' is running! Access URLs:
	Local: 		http://localhost:9999
	External: 	http://192.168.1.31:9999
	Profile(s): 	[swagger, dev]
----------------------------------------------------------
2017-02-17 16:42:12.696  INFO 1272 --- [  restartedMain] io.delverdev.microservices.SampleApp     : 
----------------------------------------------------------
	Config Server: 	Connected to the JHipster Registry config server!
----------------------------------------------------------

```
3 instances were registered (match the time in screenshot with the time in logs):
![capture](https://cloud.githubusercontent.com/assets/7104443/23069956/085010dc-f532-11e6-8dca-a073ab6007a3.PNG)
Logs of Registry when UAA registers:
```
2017-02-17 16:41:18.111  INFO 3360 --- [a-EvictionTimer] c.n.e.registry.AbstractInstanceRegistry  : Running the evict task with compensationTime 0ms
2017-02-17 16:41:21.743  INFO 3360 --- [  XNIO-2 task-2] c.s.j.s.i.a.WebApplicationImpl           : Initiating Jersey application, version 'Jersey: 1.19.1 03/11/2016 02:08 PM'
2017-02-17 16:41:22.182  INFO 3360 --- [  XNIO-2 task-2] c.n.d.provider.DiscoveryJerseyProvider   : Using JSON encoding codec LegacyJacksonJson
2017-02-17 16:41:22.183  INFO 3360 --- [  XNIO-2 task-2] c.n.d.provider.DiscoveryJerseyProvider   : Using JSON decoding codec LegacyJacksonJson
2017-02-17 16:41:22.194  INFO 3360 --- [  XNIO-2 task-2] c.n.d.provider.DiscoveryJerseyProvider   : Using XML encoding codec XStreamXml
2017-02-17 16:41:22.196  INFO 3360 --- [  XNIO-2 task-2] c.n.d.provider.DiscoveryJerseyProvider   : Using XML decoding codec XStreamXml
2017-02-17 16:41:54.378  WARN 3360 --- [  XNIO-2 task-3] c.n.e.registry.AbstractInstanceRegistry  : DS: Registry: lease doesn't exist, registering resource: SAMPLE - sample:01d89341053c58b21bce7ef55cdaa8e5
2017-02-17 16:41:54.593  WARN 3360 --- [  XNIO-2 task-3] c.n.eureka.resources.InstanceResource    : Not Found (Renew): SAMPLE - sample:01d89341053c58b21bce7ef55cdaa8e5
2017-02-17 16:41:55.554  INFO 3360 --- [  XNIO-2 task-5] c.n.e.registry.AbstractInstanceRegistry  : Registered instance SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5 with status STARTING (replication=false)
2017-02-17 16:41:57.032 DEBUG 3360 --- [  XNIO-2 task-6] i.g.j.r.s.Http401UnauthorizedEntryPoint  : Pre-authenticated entry point called. Rejecting access
2017-02-17 16:41:57.215 ERROR 3360 --- [get_localhost-9] c.n.e.cluster.ReplicationTaskProcessor   : Network level connection to peer localhost; retrying after delay

com.sun.jersey.api.client.ClientHandlerException: java.net.SocketTimeoutException: Read timed out
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:187)
	at com.netflix.eureka.cluster.DynamicGZIPContentEncodingFilter.handle(DynamicGZIPContentEncodingFilter.java:48)
	at com.netflix.discovery.EurekaIdentityHeaderFilter.handle(EurekaIdentityHeaderFilter.java:27)
	at com.sun.jersey.api.client.Client.handle(Client.java:652)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:682)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.post(WebResource.java:570)
	at com.netflix.eureka.transport.JerseyReplicationClient.submitBatchUpdates(JerseyReplicationClient.java:116)
	at com.netflix.eureka.cluster.ReplicationTaskProcessor.process(ReplicationTaskProcessor.java:71)
	at com.netflix.eureka.util.batcher.TaskExecutors$BatchWorkerRunnable.run(TaskExecutors.java:187)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:171)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at org.apache.http.impl.io.AbstractSessionInputBuffer.fillBuffer(AbstractSessionInputBuffer.java:158)
	at org.apache.http.impl.io.SocketInputBuffer.fillBuffer(SocketInputBuffer.java:82)
	at org.apache.http.impl.io.AbstractSessionInputBuffer.readLine(AbstractSessionInputBuffer.java:271)
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:140)
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:57)
	at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:259)
	at org.apache.http.impl.AbstractHttpClientConnection.receiveResponseHeader(AbstractHttpClientConnection.java:281)
	at org.apache.http.impl.conn.DefaultClientConnection.receiveResponseHeader(DefaultClientConnection.java:259)
	at org.apache.http.impl.conn.AbstractClientConnAdapter.receiveResponseHeader(AbstractClientConnAdapter.java:232)
	at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:273)
	at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:125)
	at org.apache.http.impl.client.DefaultRequestDirector.tryExecute(DefaultRequestDirector.java:686)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:488)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:884)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:117)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:173)
	... 10 common frames omitted

2017-02-17 16:41:57.976 DEBUG 3360 --- [  XNIO-2 task-7] i.g.j.r.s.Http401UnauthorizedEntryPoint  : Pre-authenticated entry point called. Rejecting access
2017-02-17 16:41:58.247 ERROR 3360 --- [et_localhost-10] c.n.e.cluster.ReplicationTaskProcessor   : Batch update failure with HTTP status code 401; discarding 1 replication tasks
2017-02-17 16:41:58.381  WARN 3360 --- [et_localhost-10] c.n.eureka.util.batcher.TaskExecutors    : Discarding 1 tasks of TaskBatchingWorker-target_localhost-10 due to permanent error
2017-02-17 16:42:04.326  INFO 3360 --- [  XNIO-2 task-8] c.n.e.registry.AbstractInstanceRegistry  : Registered instance SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5 with status STARTING (replication=false)
2017-02-17 16:42:04.928 DEBUG 3360 --- [  XNIO-2 task-9] i.g.j.r.s.Http401UnauthorizedEntryPoint  : Pre-authenticated entry point called. Rejecting access
2017-02-17 16:42:04.964 ERROR 3360 --- [et_localhost-10] c.n.e.cluster.ReplicationTaskProcessor   : Batch update failure with HTTP status code 401; discarding 1 replication tasks
2017-02-17 16:42:04.981  WARN 3360 --- [et_localhost-10] c.n.eureka.util.batcher.TaskExecutors    : Discarding 1 tasks of TaskBatchingWorker-target_localhost-10 due to permanent error
2017-02-17 16:42:12.220  INFO 3360 --- [ XNIO-2 task-11] c.n.e.registry.AbstractInstanceRegistry  : Registered instance SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5 with status UP (replication=false)
2017-02-17 16:42:13.826 ERROR 3360 --- [et_localhost-16] c.n.e.cluster.ReplicationTaskProcessor   : Network level connection to peer localhost; retrying after delay

com.sun.jersey.api.client.ClientHandlerException: java.net.SocketTimeoutException: Read timed out
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:187)
	at com.netflix.eureka.cluster.DynamicGZIPContentEncodingFilter.handle(DynamicGZIPContentEncodingFilter.java:48)
	at com.netflix.discovery.EurekaIdentityHeaderFilter.handle(EurekaIdentityHeaderFilter.java:27)
	at com.sun.jersey.api.client.Client.handle(Client.java:652)
	at com.sun.jersey.api.client.WebResource.handle(WebResource.java:682)
	at com.sun.jersey.api.client.WebResource.access$200(WebResource.java:74)
	at com.sun.jersey.api.client.WebResource$Builder.post(WebResource.java:570)
	at com.netflix.eureka.transport.JerseyReplicationClient.submitBatchUpdates(JerseyReplicationClient.java:116)
	at com.netflix.eureka.cluster.ReplicationTaskProcessor.process(ReplicationTaskProcessor.java:71)
	at com.netflix.eureka.util.batcher.TaskExecutors$BatchWorkerRunnable.run(TaskExecutors.java:187)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:171)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at org.apache.http.impl.io.AbstractSessionInputBuffer.fillBuffer(AbstractSessionInputBuffer.java:158)
	at org.apache.http.impl.io.SocketInputBuffer.fillBuffer(SocketInputBuffer.java:82)
	at org.apache.http.impl.io.AbstractSessionInputBuffer.readLine(AbstractSessionInputBuffer.java:271)
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:140)
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:57)
	at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:259)
	at org.apache.http.impl.AbstractHttpClientConnection.receiveResponseHeader(AbstractHttpClientConnection.java:281)
	at org.apache.http.impl.conn.DefaultClientConnection.receiveResponseHeader(DefaultClientConnection.java:259)
	at org.apache.http.impl.conn.AbstractClientConnAdapter.receiveResponseHeader(AbstractClientConnAdapter.java:232)
	at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:273)
	at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:125)
	at org.apache.http.impl.client.DefaultRequestDirector.tryExecute(DefaultRequestDirector.java:686)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:488)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:884)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:117)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at com.sun.jersey.client.apache4.ApacheHttpClient4Handler.handle(ApacheHttpClient4Handler.java:173)
	... 10 common frames omitted

2017-02-17 16:42:13.847 DEBUG 3360 --- [ XNIO-2 task-13] i.g.j.r.s.Http401UnauthorizedEntryPoint  : Pre-authenticated entry point called. Rejecting access
2017-02-17 16:42:14.212 DEBUG 3360 --- [ XNIO-2 task-14] i.g.j.r.s.Http401UnauthorizedEntryPoint  : Pre-authenticated entry point called. Rejecting access
2017-02-17 16:42:14.295 DEBUG 3360 --- [ XNIO-2 task-15] i.g.j.r.s.Http401UnauthorizedEntryPoint  : Pre-authenticated entry point called. Rejecting access
2017-02-17 16:42:14.309 ERROR 3360 --- [get_localhost-9] c.n.e.cluster.ReplicationTaskProcessor   : Batch update failure with HTTP status code 401; discarding 1 replication tasks
2017-02-17 16:42:14.334  WARN 3360 --- [get_localhost-9] c.n.eureka.util.batcher.TaskExecutors    : Discarding 1 tasks of TaskBatchingWorker-target_localhost-9 due to permanent error
2017-02-17 16:42:18.111  INFO 3360 --- [a-EvictionTimer] c.n.e.registry.AbstractInstanceRegistry  : Running the evict task with compensationTime 0ms
```
Then every 30 seconds:
```
2017-02-17 16:42:26.617 DEBUG 3360 --- [  XNIO-2 task-1] i.g.j.r.s.Http401UnauthorizedEntryPoint  : Pre-authenticated entry point called. Rejecting access
2017-02-17 16:42:26.635 ERROR 3360 --- [et_localhost-10] c.n.e.cluster.ReplicationTaskProcessor   : Batch update failure with HTTP status code 401; discarding 1 replication tasks
2017-02-17 16:42:26.647  WARN 3360 --- [et_localhost-10] c.n.eureka.util.batcher.TaskExecutors    : Discarding 1 tasks of TaskBatchingWorker-target_localhost-10 due to permanent error
```
Logs of UAA (they repeat):
```
2017-02-17 16:42:12.762 DEBUG 1272 --- [nfoReplicator-0] s.n.www.protocol.http.HttpURLConnection  : sun.net.www.MessageHeader@70eec3256 pairs: {GET /config/application/dev/master HTTP/1.1: null}{Accept: application/json, application/*+json}{Authorization: Basic YWRtaW46YWRtaW4=}{User-Agent: Java/1.8.0_121}{Host: localhost:8761}{Connection: keep-alive}
2017-02-17 16:42:14.209 DEBUG 1272 --- [nfoReplicator-0] s.n.www.protocol.http.HttpURLConnection  : sun.net.www.MessageHeader@9954f0411 pairs: {null: HTTP/1.1 200 OK}{Expires: 0}{Cache-Control: no-cache, no-store, max-age=0, must-revalidate}{X-XSS-Protection: 1; mode=block}{Pragma: no-cache}{Date: Fri, 17 Feb 2017 14:42:14 GMT}{Connection: keep-alive}{X-Content-Type-Options: nosniff}{Transfer-Encoding: chunked}{Content-Type: application/json;charset=UTF-8}{X-Application-Context: jhipster-registry:native,dev:8761}
2017-02-17 16:42:20.363 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.MonitoredConnectionManager  : Closing connections idle longer than 30000 SECONDS
2017-02-17 16:42:20.382 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.NamedConnectionPool         : Closing connections idle longer than 30000 SECONDS
2017-02-17 16:42:20.628 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.MonitoredConnectionManager  : Closing connections idle longer than -1 SECONDS
2017-02-17 16:42:20.646 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.NamedConnectionPool         : Closing connections idle longer than 0 SECONDS
2017-02-17 16:42:20.646 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.NamedConnectionPool         : Closing connection last used @ Fri Feb 17 16:41:54 EET 2017
2017-02-17 16:42:20.667 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.NamedConnectionPool         : Deleting connection [{}->http://localhost:8761][null]
2017-02-17 16:42:20.669 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.NamedConnectionPool         : Closing connection last used @ Fri Feb 17 16:42:12 EET 2017
2017-02-17 16:42:20.690 DEBUG 1272 --- [t-Conn-Cleaner2] c.n.d.shared.NamedConnectionPool         : Deleting connection [{}->http://localhost:8761][null]
2017-02-17 16:42:25.997 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.MonitoredConnectionManager  : Get connection: {}->http://localhost:8761, timeout = 5000
2017-02-17 16:42:26.031 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : [{}->http://localhost:8761] total kept alive: 0, total issued: 0, total allocated: 0 out of 200
2017-02-17 16:42:26.031 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : No free connections [{}->http://localhost:8761][null]
2017-02-17 16:42:26.046 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Available capacity: 50 out of 50 [{}->http://localhost:8761][null]
2017-02-17 16:42:26.046 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Creating new connection [{}->http://localhost:8761]
2017-02-17 16:42:26.114 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.MonitoredConnectionManager  : Released connection is reusable.
2017-02-17 16:42:26.114 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Releasing connection [{}->http://localhost:8761][null]
2017-02-17 16:42:26.124 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Pooling connection [{}->http://localhost:8761][null]; keep alive indefinitely
2017-02-17 16:42:26.125 DEBUG 1272 --- [tbeatExecutor-0] c.n.d.shared.NamedConnectionPool         : Notifying no-one, there are no waiting threads
2017-02-17 16:42:26.125 DEBUG 1272 --- [tbeatExecutor-0] n.d.s.t.j.AbstractJerseyEurekaHttpClient : Jersey HTTP PUT http://admin:admin@localhost:8761/eureka//apps/SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5; statusCode=200
2017-02-17 16:42:26.125 DEBUG 1272 --- [tbeatExecutor-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_SAMPLE/sample:01d89341053c58b21bce7ef55cdaa8e5 - Heartbeat status: 200
2017-02-17 16:42:29.765  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Disable delta property : false
2017-02-17 16:42:29.772  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Single vip registry refresh property : null
2017-02-17 16:42:29.774  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Force full registry fetch : false
2017-02-17 16:42:29.774  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Application is null : false
2017-02-17 16:42:29.784  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Registered Applications size is zero : true
2017-02-17 16:42:29.784  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Application version is -1: false
2017-02-17 16:42:29.784  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Getting all instance registry info from the eureka server
2017-02-17 16:42:29.793 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.MonitoredConnectionManager  : Get connection: {}->http://localhost:8761, timeout = 5000
2017-02-17 16:42:29.799 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.NamedConnectionPool         : [{}->http://localhost:8761] total kept alive: 1, total issued: 0, total allocated: 1 out of 200
2017-02-17 16:42:29.799 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.NamedConnectionPool         : Getting free connection [{}->http://localhost:8761][null]
2017-02-17 16:42:29.845 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.MonitoredConnectionManager  : Released connection is reusable.
2017-02-17 16:42:29.845 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.NamedConnectionPool         : Releasing connection [{}->http://localhost:8761][null]
2017-02-17 16:42:29.845 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.NamedConnectionPool         : Pooling connection [{}->http://localhost:8761][null]; keep alive indefinitely
2017-02-17 16:42:29.845 DEBUG 1272 --- [freshExecutor-0] c.n.d.shared.NamedConnectionPool         : Notifying no-one, there are no waiting threads
2017-02-17 16:42:29.845 DEBUG 1272 --- [freshExecutor-0] n.d.s.t.j.AbstractJerseyEurekaHttpClient : Jersey HTTP GET http://admin:admin@localhost:8761/eureka//apps/?; statusCode=200
2017-02-17 16:42:29.846  INFO 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : The response status is 200
2017-02-17 16:42:29.847 DEBUG 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Got full registry with apps hashcode STARTING_1_
2017-02-17 16:42:29.847 DEBUG 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : The total number of all instances in the client now is 1
2017-02-17 16:42:29.849 DEBUG 1272 --- [freshExecutor-0] com.netflix.discovery.DiscoveryClient    : Completed cache refresh task for discovery. All Apps hash code is Local region apps hashcode: STARTING_1_, is fetching remote regions? false 
```
##### **Reproduce the error**

1. Clone Registry
2. Generate UAA
3. Start Registry via IDE
4. Start UAA via IDE

##### **Related issues**

[Cannot start UAA server](https://github.com/jhipster/generator-jhipster/issues/5215)  
[Unable to deploy a simple microservice to Heroku (90sec timeout)](https://github.com/jhipster/generator-jhipster/issues/5190)
##### **JHipster Version(s)**

```
D:\microservices\uaa
`-- generator-jhipster@4.0.3

```


##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**  
UAA
```yaml
{
  "generator-jhipster": {
    "jhipsterVersion": "4.0.3",
    "baseName": "sample",
    "packageName": "io.delverdev.microservices",
    "packageFolder": "io/delverdev/microservices",
    "serverPort": "9999",
    "authenticationType": "uaa",
    "hibernateCache": "hazelcast",
    "clusteredHttpSession": false,
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "mysql",
    "prodDatabaseType": "mysql",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": "eureka",
    "buildTool": "gradle",
    "enableSocialSignIn": false,
    "enableTranslation": false,
    "applicationType": "uaa",
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "skipClient": true,
    "clientPackageManager": "yarn"
  }
}
```


##### **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

ls: no such file or directory: .jhipster/*.json

##### **Browsers and Operating System**

java version "1.8.0_121"
Java(TM) SE Runtime Environment (build 1.8.0_121-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.121-b13, mixed mode)

git version 2.11.1.windows.1

node: v6.9.5

npm: 3.10.10

yarn: 0.19.1

- [x] Checking this box is mandatory (this is just to show you read everything)

--- a/generators/server/templates/gradle/_profile_dev.gradle
+++ b/generators/server/templates/gradle/_profile_dev.gradle
@@ -55,6 +55,8 @@ processResources {
     <%_ } _%>
         filter {
             it.replace('#spring.profiles.active#', profiles)
+        }
+        filter {
             it.replace('#project.version#', version)
         }
     }

--- a/generators/server/templates/gradle/_profile_prod.gradle
+++ b/generators/server/templates/gradle/_profile_prod.gradle
@@ -57,6 +57,8 @@ processResources {
     <%_ } _%>
         filter {
             it.replace('#spring.profiles.active#', profiles)
+        }
+        filter {
             it.replace('#project.version#', version)
         }
     }

